src/CLI/CommandLineParser.idr

```diff
@@ -4,6 +4,23 @@ import Options.Applicative
 
 %default total
 
+public export
+data InputFormat =
+
+  ||| Lisp syntax is the traditional lisp-like syntax for Typedefs
+  Lispy |
+
+  ||| IndexFree syntax is the new Typedefs syntax
+  ||| That looks more like ml-style data declaration
+  ||| It has some limitations compared to lispy, but is much more
+  ||| familiar to use.
```

Comment:
```suggestion
  ||| IndexFree syntax is the new Typedefs syntax that looks more like ML-style data declarations.
  ||| It has some limitations compared to the lispy one, but is much more familiar to use.
```

Reply:
Suggestion: shouldn't we call it `MLStyle` instead then?

Reply:
I guess? I feel like the ML-style is the marketing term while the original intent for it was to avoid having to manually track debrujin indices

Reply:
I've renamed it ML style since it was either that, or rename the other "SExpr"